ragaai_catalyst/tracers/agentic_tracing/upload/upload_local_metric.py

```
@@ -68,5 +68,5 @@ def calculate_metric(project_id, metric_name, model, provider, prompt, response,
         response.raise_for_status()
         return response.json()
     except requests.exceptions.RequestException as e:
-        logger.debug(f"Error in calculate-metric api: {e}")
-        raise e
+        logger.debug(f"Error in calculate-metric api: {e}, payload: {payload}")
+        raise Exception(f"Error in calculate-metric: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)